### PR TITLE
chore(package): update typescript to version 3.9.6

### PR DESCRIPTION
**Pull request by ruscon**
\+ an example of using the new typescript functionality "?" in code

### CODE REVIEW

File: src/entity-manager/EntityManager.ts [hunk 1]
```diff
@@ -161,17 +161,17 @@ export class EntityManager {
     /**
      * Creates a new query builder that can be used to build a sql query.
      */
-    createQueryBuilder<Entity>(entityClass: ObjectType<Entity>, alias: string, queryRunner?: QueryRunner): SelectQueryBuilder<Entity>;
```
Comment (pleerock) [inline src/entity-manager/EntityManager.ts:164]: I don't think we should have `= any` here. What is the motivation behind this change?

File: src/query-builder/SelectQueryBuilder.ts [hunk 2]
```diff
@@ -181,21 +181,21 @@ export class SelectQueryBuilder<Entity> extends QueryBuilder<Entity> implements
      * Also sets a main string alias of the selection data.
      * Removes all previously set from-s.
      */
-    from<T>(entityTarget: (qb: SelectQueryBuilder<any>) => SelectQueryBuilder<any>, aliasName: string): SelectQueryBuilder<T>;
+    from<T = any>(entityTarget: (qb: SelectQueryBuilder<any>) => SelectQueryBuilder<any>, aliasName: string): SelectQueryBuilder<T>;
```
Comment (pleerock) [inline src/query-builder/SelectQueryBuilder.ts:184]: same comment here

File: src/entity-manager/EntityManager.ts (hunk 1 above)
Comment (ruscon) [reply src/entity-manager/EntityManager.ts:164]: > I don't think we should have `= any` here. What is the motivation behind this change?

Yes, I think it should be unknown, but many tests fall

Comment (pleerock) [reply src/entity-manager/EntityManager.ts:164]: I don't think we should put a default type at all, what can be a motivation? It looks like a monkey patching instead of root problem resolving solution.

Comment (ruscon) [reply src/entity-manager/EntityManager.ts:164]: In the latest typescript version, for example in this block `createQueryBuilder<Entity>`, `Entity` by default will be `unknown` (in the current version used it is interpreted as `any`)
You can just upgrade the typescript version and see the difference
